ozaria/site/components/interactive/PageInteractive/insertCode/index.vue

```diff
@@ -0,0 +1,209 @@
+<script>
+  import { codemirror } from 'vue-codemirror'
+
+  // TODO dynamically import these
+  import 'codemirror/mode/javascript/javascript'
+  import 'codemirror/mode/python/python'
+  import 'codemirror/lib/codemirror.css'
+
+  import BaseInteractiveTitle from '../common/BaseInteractiveTitle'
+
+  const SAMPLE_CODE = `
+          let x = "y"
+          # This is the line to replace
+          let a = "b"
+          console.log(x + a)
+          `
+  // This is 1 indexed. The first line is 1 not 0.
+  const LINE_TO_REPLACE = 4
```

Comment:
Looks like it should be 2 as per the sample code above.

Reply:
Is it fine to leave as it will quickly be replaced by real data? Was testing an edge case.

Also thanks for pointing it out!

Reply:
Yep that seems fine 👍